bundles/org.openhab.ui/web/src/pages/settings/rules/script/blockly-editor.vue

```diff
@@ -318,7 +319,7 @@
         </block>
       </category>
 
-      <category name="openHAB" colour="0">
+      <category name="openHAB" colour="0" :expanded="$f7.device.desktop">
```

Comment:
This should only be expanded on desktops so using Framework7 [Device](https://v5.framework7.io/docs/device) property available under the `this.$f7` in Vue components.